.gitignore

```diff
@@ -72,3 +74,27 @@ bin/waagent2.0c
 
 # mac osx specific files
 .DS_Store
+
```

Comment:
wouldn't it be better to move your virtual environment out of the agent's folder?

in case we ever need to add any of those paths to the agent

Reply:
It's pretty common to set up a virtual environment in the folder your project is in, I think. I was hitting this same issue but I gave up on doing `git add .` and instead try to add everything except venv.

Reply:
I setup my virtual environments outside source control and share them between projects.

Maybe you guys can set yours within a subdirectory so that we do not need to filter out bin/lib/include/scripts (which are very common and we may use in the future) at the root level?

Reply:
@narrieta: I agree that setting up VirtualEnv outside is the ideal way. But pycharm sets it up in the folder. I did fix it for my dev-env.

But, this PR was to keep the `.gitignore` in such a way that it helps any other developers to not send `venv` by mistake.